schema_based_on_diagram.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name varchar(25) NOT NULL,
+  date_of_birth DATE
+)
+
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP DEFAULT NOW(),
+  patient_id int REFERENCES patients(id),
+  status varchar(25) NOT NULL
+)
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY,
+  type varchar(25) NOT NULL,
+  name varchar(25) NOT NULL  
+)
+
+CREATE TABLE medical_treatments (
+  id SERIAL PRIMARY KEY,
+  medical_histories_id int REFERENCES medical_histories  (id),
+  treatments_id int REFERENCES treatments(id)
+)
+
+CREATE TABLE invoices (
+  id SERIAL PRIMARY KEY,
+  total_amout DECIMAL,
+  generated_at TIMESTAMP DECIMAL NOW(),
+  payed_at TIMESTAMP DECIMAL NOW(),
+  medical_history_id int REFERENCES medical_histories(id)
+)
+
```

Comment:
- [ ] Your project is complete, just missing one thing is to add indexes to all FOREIGN KEYS as the requirement says `Remember to add the FK indexes.`.

Something like that, but add for all
`CREATE INDEX ON medical_histories (patient_id);`